sdk/identity/azure-identity/out/azure/identity/_credentials/environment.pyi

```diff
@@ -0,0 +1,15 @@
+# Stubs for azure.identity._credentials.environment (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from .._constants import EnvironmentVariables
+from .client_credential import CertificateCredential, ClientSecretCredential
+from .user import UsernamePasswordCredential
+from azure.core.credentials import AccessToken
+from typing import Any, Mapping
+
+EnvironmentCredentialTypes: Any
+
+class EnvironmentCredential:
+    def __init__(self, **kwargs: Mapping[str, Any]) -> None: ...
```

Comment:
Is there a reason our kwargs types vary so much?  (Mapping[] vs Any)  If it matters at all, I'd bang the consistency hammer again, but if this is premature ignore moe.

Reply:
Accidents of history, I'll make them consistently `Any` when I do a mypy pass.